Karma/Pages/Requests.cshtml.cs

```diff
@@ -37,9 +37,9 @@ public RequestsModel(IRequestRepository requestService, IMessageRepository messa
             MessageService = messageService;
         }
 
-        public void OnGet()
+        public async void OnGet()
         {
-            Requests = RequestService.SearchPosts(SearchTerm);
+            Requests = await RequestService.SearchPosts(SearchTerm);
         }

```

Comment:
```suggestion
            Requests = await RequestService.SearchPosts(SearchTerm);
            Return Page();
        }
```
Needs a return statement. (2 - github couldn't let me multi-line edit, so this is the second part of the same comment)